Breeze.UI/src/app/wallet/tumblebit/tumblebit.service.ts

```diff
@@ -6,28 +6,25 @@ import 'rxjs/add/operator/mergeMap';
 import 'rxjs/add/operator/take';
 import 'rxjs/add/operator/concat';
 import 'rxjs/add/observable/throw';
-
 import { TumblerConnectionRequest } from './classes/tumbler-connection-request';
 import { TumbleRequest } from './classes/tumble-request';
 import { ConnectRequest } from './classes/connect-request';
 import { GlobalService } from '../../shared/services/global.service';
+import { Error } from '../../shared/classes/error';
 
 @Injectable()
 export class TumblebitService {
+
   // The service to connect to & operate a TumbleBit Server via the
   // TumbleBit.Client.CLI tool
   private headers = new Headers({ 'Content-Type': 'application/json' });
-
   private pollingInterval = 3000;

```

Comment:
If it is a quick think to do it would be good to use `pollingInterval` from **api.service.ts**